mRides-app/Resources/layout/UserProfile.axml

```diff
@@ -35,11 +35,35 @@
                 android:layout_height="89.7dp"
                 android:textColor="#161616"
                 android:textSize="23dp" />
-            <ImageView
-                android:src="@android:drawable/ic_menu_gallery"
-                android:layout_width="55.0dp"
-                android:layout_height="45.0dp"
-                android:id="@+id/genderImage" />
+            <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/linearLayout1"
```

Comment:
@AlineTheAlien I see some ids that are too generic, it might create conflicts in the future